### PR TITLE
Added new save method 'SYNCHRONOUS_DML'

### PR DESCRIPTION
**Pull request by jongpie**
This skips publishing platform events and instead immediately creates Log__c and LogEntry__c records

### CODE REVIEW

File: nebula-logger/main/log-management/classes/LogEntryEventHandler.cls [hunk 1]
```diff
@@ -36,13 +37,13 @@ public without sharing class LogEntryEventHandler {
      * @description Runs the trigger handler's logic for the `LogEntryEvent__e` platform event object
      */
     public void execute() {
-        switch on this.triggerOperationType {
-            when AFTER_INSERT {
-                this.upsertLogs();
-                this.insertLogEntries();
-                this.insertTopics();
-            }
+        if (this.logEntryEvents.isEmpty()) {
```
Comment (jamessimone) [inline nebula-logger/main/log-management/classes/LogEntryEventHandler.cls:40]: something that I think helps with the readability/avoidance of an early return is inverting the conditional - if they list is _not_ empty, upsert logs, insert log entries, etc ...

**Comment by jamessimone** [comment]
@jongpie one minor comment, this looks great!

File: nebula-logger/main/log-management/classes/LogEntryEventHandler.cls (hunk 1 above)
Comment (jongpie) [reply nebula-logger/main/log-management/classes/LogEntryEventHandler.cls]: You've convinced me! I've flipped it around so it calls the private methods if the list is not empty.

**Comment by jongpie** [comment]
@jamessimone thanks for reviewing!

**Review by jamessimone — APPROVED** [review]

**Pull request merged**